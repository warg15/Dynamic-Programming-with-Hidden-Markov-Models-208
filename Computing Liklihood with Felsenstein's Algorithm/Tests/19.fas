>White-throated-tinamou
TGGTACGCTACCTGTGAAGCCCCTGCATCTACCAACGCGCCAAGTACCGGTGTGATATGGCTCTGCATCAACTCGCTACTTATTTACAAGACTTTAGATGACTAGGCCTTTGCTTACTTATGAATTGCTGATTAGAGTACGACAGTTGCGATAATATGCATTACGAATTTAAGAGCCAAGCACATCATTACGCTGTAGTG
>Emperor-penguin
TGGTATGCCATCTGTGGGAGCCCTCTAATTATCAACGCGCTGAGTGCCGGGACGCTCTGTCGCCGCAGCAACTAGCAATTCATTTACAAGGCTTTAGATTGCTGGGTTTGTGCTTGCTCATGAATTGCTGATTTGTATGCGACAGTCGCGATAGTATGCATGACGGGTTTAATAGCCACGAACATCGTAAGACCGCAGTG
>Turkey
TGGTATCGGCTCTGCGCGAGCCCTCTAAATATCGACGCGCCGAGTGCGCGGACGATACGACGCTGCAGCAACTAGCAACTCATTTACATGGCTTCAGATTTCTGGGTTTGCTCTGGTTCATGAATCGCTGATTTGAATGCGACAGTCACGATAGTATACGTGTTGTGTCTAAGAACCAAGAACAACGTAAGCCCGCAGTG
>Carmine-bee-eater
TGGTATGTGATCCATAGTAGCCCTCCAATTATCAAATCGCCGAGTGCCCGAACGATTTGTCGCCTCAGCAACTAGCAACTCGTTTACACGGCTTGTGATCGCTGGGTTTTTGTTTGCCCATGATTTACGGATTTGTCTGCAACAGTCGCTATAGTATGCATGACGGGTTTAACAGTCAAAAACATCGTAAGAGCGCAGTG
>Hoatzin
TGGTATGCGATCTGTGGGGGCCCTCTAATTATCAACGCGCTGAGTGCCGGGACGATCTGTGGCCGCCGCAACTAGTAATTCATTTACAAGGCTTTAGATCGCTGGGTTTGCGCTTGGTCATGAATTTCTGATTTGTATGCGACAATCGCGATAGTATGCATGACAGGTTTTACAGCCAAGGACATCGTAAGACCGCAGTG
>Brown-mesite
TGGTATGCGACCTGTGAGAGCACTCTAATTATCAACCCGCCGAGTGCAGGGACGATTCGGCGTCGCAGCAACTAGCTATTCACTTACAAGGCTTTAGATTGCTGGGTTTGTGCTTGCTGATGAATTGCTGATTTGTATACGACGGTCGCGATAGTATGCAGGACGGGTTTAACAGTCAGGATCATCGTAAGACCGCAGTG
>Chicken
TGGTATCCGCCCTGCGGGAGCCCTCTAAATATCGACGCGCCGAGTGCGCGGACGATACGTCGCTGCTGCAACTAGCAACTCATTTACATGGCTTCAAATTTCTGGGTTTGTTCTGGGCCATGAATTGCTGATTTGAATGCGACAGTCACGATAGTAGACGTGCTGGGTCTAAGAACCAAGAACAACGTAAGCCCGCAGTA
>Rifleman
TAGTCTGCGATTTATGTGAGCCCTCTGATTATCAACTCGCCGAGTGCCGGGACTATCTGTCACCGCAGCAACTAGCAATTCATTTACGAGCCTTCAGATTGCAGGGGTGGAGCTTGCTCATGAATTGCTGATTTGTCAGCGACAGTCGGGATAGTATACATGACGGGTTTAACAGCCAAGAACACCTCAAGACCGCAATG
>Pekin-duck
TGGCGTGCGCTCTGTGGGAGCCCTCTAATTATCAACGCGCCGAGTACCGGGACGAGCTGGCGCTGCAGCAACTAGCAACTTATTTACAAGGCTTTAGATTGCTGGGTTTGTGCTGGCTTACGAATTGCTGAGTTGAATGCGACAGTCGCGATAGTATGCGTTACGGGTTTAAGAGCCCAGAACATCGCAAGGCCGCAGTA
>Chimney-swift
CGGTATGCCATCTGTGCGCGCACTCCAATAATCAACCCGCCGAGTGCCGGGACGATCTGTCGCCGCAGCAACTAGCAATTCATTTGCAAGGCTCTAGATTGCAGGGTCTGTGCGTGCTAATGAATTGCTGATTGATATGCGGAAGTCGCGAGAGCATGCATGACGGACCTAACAGCGAGGACCGTCGTAAGACTGCAATG
>Peregrine-falcon
TGGTATGCGATCTGTGGGAGCCCTCTAATTATCAACGCGCCGAGTGCGGGGATGATCTGTCGCCGCAGCAACTAGCAACTCATTTGCAAGGCATTAGATTGCTGGGTTTGTGCTTGCCCATGAATTGCTGGTTTGTCTGCGACAATCGCGACAGTACGCATGACGGGTTTAACAGCTAAGAACATCGTAAGACTGCAGTG
>Crested-ibis
TGGTATGCGATCTGTGGGAGCCCTCTAATTATTAACGCGCCGAGTGCTGGGACGATCTGTCGCCGCAGCAACTAGCAATTCATTTACAAGGCTTTAGATTTCTGGGTTTGTGCTTGCTCATGAATTGTTGATTTGTATGCGACAGTCGCGATAGTATGCATGACGGGTTTAACAGCCAAGAGCATCGCAAGACCGCAGTG
>White-tailed-tropicbird
TGGTATGCGATCTGTGGGAGTCTTCTAATTATCAACGCGCCGAGTGTCGGGATGATTTGTCGCCGCAGCAACTAGCAATTCATTTACAAGGCCTTAGATTGCTGGATTTGTGCTTCCTCACGAGTTGCTGATTTGTATGCGACAGTCGCGATAGTATACTTGACGGGTTTAACAGCAAAGATCATCGTAAGACCGCAGTG
>Northern-fulmar
TGGTATGCGATCTGTGGGAGCCCTCTAATTGTCAACGCGCCGAGTGCCGGGACGATCTGTCTCCGCAGCAACTAGCAATTCATTTACAAGGCTTTAGATTGCTGGGTTTGTGCTTGCTCATGAATTGCTCATTTGTATGCGCCAGTCGCGATAGTATGCATGACGGGTTTAACAACCAAGAACATCGTAAGACCGCAGTG
>Pigeon
TGGTATGCGATCTGTGGGAGCCCTCTAATTATCAACGCGCTGAGATCCGCGACGATCTGACGCCGCAACAACTAGCAACTTATTTACAAGGCTTTAGGTTGCTAGGCTTGTGCTTGCTGATGAATTGCTGATTTATATGCGACAGTCGCGACAATATGTATGACAGGTTTAACAGCCAAGAACATCGTAAGAGCGCAGTG
>Budgerigar
TAGTATGCGATCTGTGGGAGCCTCCTAATTGTAAATGCCCGGAGTGCCGGGACGATCCGTCGCCGCAGCAACTAGTAATTTATCTGCGAGGATTAAGTTTGATGGGTTGGTTCTTGCTCGTGAATTGTTCATTTGTCTGCGACAGTCGCGGTAATATGCTTGACGGGTCTGACAGCCAAGAACGTCGCAAGACCGCAGTG
>Golden-collared-manakin
TAATTTGCGATCCGTGGGAGCCCTATAATGATTAACGCGTTGAGTGCCGAGACGATCTGGCGCCGCCGCAACTAGCAATTCATTTACGAGCCTTTAGCTTGCTGGGATGGGGCTTGCTCACGAATTGCTGCTTTGCCAGGGACAGTCGGGATAGTATGCATGACGGGTTTAGCAGCCAAGAACATCGTAAGACGGCAGTG
>Barn-owl
TGGTATGCGGTCTGGGAAAGCCCTCTAATTATCAAAGCGCTGAGTGCCGTGACGAGCTGTCGCCGTAGCAATTAGTAATTCATTTACAAGGCCTGAAATTGCTGTGTTTGTGCTTGATCATAAATTGCTGATTCGTCTGCTACAGTCGGGATGGTATGCATGACGGGTTTAACAGCCAAGAACATCGTAAGACCGCAGTG
>American-flamingo
TGGTATGCAATCTGTGGGAGCTCTCTATTTATCAACTCGCCGAGTGCCGGGACGATCTGGCGCCGCAGCAACTAGCAATTCATTTACAAGACTTTAGATTGCTGGGTTTGTGCTTGCTCATGAATCGCTGAGTTATATGCGACAGGCGCGATAGTATGCATGACGGGTTTAACAGCCAAGAATATCGTAAGACCGCAGTG
>Speckled-mousebird
TGGCACGTGATCTGTGGAACCCCTCTAATTCTCAACGCGTCGAGTTCCGGGACGATCTATCGGCGCAGCCACTAGCAATTCATTTACGACGCGTTAGATTGCCGAGTTTGTGCTTATTCATGAATAGCTGATTTGTCTGCGTCAGTCGCGATAGTATGCATGACGTGTTTGACAGCCAAGAACATTGTGAGACCGCAGTG
>Little-egret
TGGTATGTGATATGTGGGAGCCCTCTAGTTATCAACGAGCCGAGTGCCGGGACGATCTGTCGCGGCAGCAGCTAGCAATTCATTTACAAGGCTTTAAATTGCTGGGTTTGTGCTTGCTCATGAATTGCTGATTAGTATGCGACCGCCGCGATAGTATGCATGACGGGTTTAACAGCCAAGAACATCGTAAGACCACAGTG
>Turkey-vulture
TGGTATGCGATCTGTGGGAGCCCTCTAATTATCAACGCGCCGAGTGCCGGGACGATCTGTCGCCGCAGCAACTGGCGATTCATTTACAATGGTTGAGATTGCTGGGTTTGTGCTTGCTCATGAATTGCTGATTTGTCTGCGACAGTCGCGATAGTATGCATGACGGGTTTAACAGCCAAGAACATCGTAAATCCGCAGTG
>Red-crested-turaco
TGGTATGCGATCTGTGGGAGCCCTCCAATTATCAACGCGCCGAGCGCCGGAACGAACTGTCACCGCAGCAACTAGCACTTCATTTACAAGGCTTTAGATTGCTGTGTTTGGGCTTGCTCATGAATTGCTGATTGGTATGCGACAGTCGCGATAGTATGCGTGACGGGTTTATCAGCCAAGAATATCTTAAGACCGCAGAG
>Rhinoceros-hornbill
TAGTATGCGATCTGTGGAAGCCCCCTGATTATCGACGCGAGGAGTGCCGGGATAATTTGTCGTCGCAGCAACTAGCAATTCATTTACAAGGCTTGCGATCCCTGGGTTTGTGTTTGCTCATGAATCTCTGATTTGTCTGCGACAATCGCGATAGTACGCATGACGGATTTAATAGCCAATAACATCTGGAGACCCCAGTG
>Grey-crowned-crane
TGGTGTGCGATCTGTGGGAGCCCTCTAATTATCAACGCGCCGAGTGCCGGGACGATCTGTCGCCGCAGCAACTAGCAATTCATTTACAAGGCCTTAGCTTGCTGGGTTTGTGCTTGCTCAAGAATTGCTGATTTGTATGCGACAGTAGCGATAGTATGCATGACGGGTTTAACAGCCAAGAACATCGTAAGACCGCAGTG
>Common-ostrich
TGGTATGCGACTTGTGAGAGCCCTGCATCTATCAACGCGCCAAGTACCGGGGCAATACGGCACCGCATCAACTCGCAACTCATTTGCAAGACTTTAAATGACTGGCTTTGTGCTTGTACAAGAATTGCTCATCAGAGCGCGACAGTCGCGATAGTATGCTTAACAGCTTTAAGAGCCAAAAACATCGTAAGGCAGTAGTG
>American-crow
TAGTTTGCGATTTGTGGGAGCCCTATAATTATCGACGCGCCGAGTGCCGGGACGATCTGTCGCCTCAACAACTAGCGATTCATTGACGAGCCTTTAGATTGCTGGGGTGGGGCTTGTTCATGAATTGCTGATTTGTAATCGACAGTCGGGATAGTATGCATGACGGCTTTAACAGCCAAGAACATCGTAAGAACGTAGTG
>White-tailed-eagle
TGATATGCGATCTGTGGGCGCCCTCTAATTATCAACCCGCCGAGTGCCGGGACGATCTGTCCCCGCAGCAACTAGCAATTCATTTACAAGACTTGAGATTGCTATGTTTGTGCTTGCTCATGAATTGCTGATTTGTCTGCGGCAGTCGCGATAGTATGCATAACGGGTTTAACAGCCAACAACATCGTAAGACCGCAGTG
>Red-legged-seriema
TGGTATGCGATCTGTGGGAGCCTTCTAATTATCAATGCGCCGAGTGCCGGGACGATCTACCGCCGCAGCAACTAGCGATTCATTTACAAGCCTTTAGATTGCTGGGTTTGTGCTTGCTCCTGAATTGCTGATGTGTCAGCGACAGTCGCGATAGTATGCGTGACGGGTTTAACAGTCAAGAACATCGTAAGACTCCAGTG
>Dalmatian-pelican
AGGTATGCGATCTCTGGGAGCCGTCTAATTATCAATGCGCCGAGTACCGGGACGATCTGTCGTCGCATCAACTAGCAATTCATTTACAAGGCTTTAGATTGATGGGTTTGTGCTTGCTCACGAATTGCTGATTTGTATGCGACAGTCGCGATAGTATGCATGACGGGTTTAACAGCCAAGAACATCGTAAGAACGCAGTG
>Adelie-penguin
TGGTATGCGATCTGTGGGAGCCCTCTAATTATCAACGCGCTGAGTGCCGGGACGATCTGTCGCCGCAGCAACTAGCAATTCATTTACAAGGCTTTAGATTGCTGGGTTTGTGCTTGCTCATGAATTGCTGATTTGTATGCGACAGTCGCGATAGTATGCATGACGGGTTTAATAGCCAAGAACATCGTAAGACCGCAGTG
>Zebra-finch
TAGTTTGCGATCTATGGGAGCCCTATAATTATCAACGCGCCGAGGGCCGGGACGATTTGTCGCCGCAGCAACTAGCGATTCATTGACGAGCCTTTAGATTGCTGGGGTGGGATTTGCTCATGAACTGCCGATTTGTCACCGACAGTCGGGATGGTATACATGTCGGATTCAACAGCCGAGAAAATGGTAGAACCGTAGTG
>Annas-hummingbird
TGGTATGCGTTCTGTGCCCGCCCTCCAATTATCGACGCGCCGAGTGCCGGGATGATCTGTCGCCGCAGTAACTAGCGATTCATTTACAACGCTCTAGATTACTGAGTTTGTGTTTTCTCATGAACTGCTGATTGATATACGACAGCCCCGATAGTATGCATAACGGGTTTGATAGTCAAGGACACCGTAAGTCCGCAATG
>Killdeer
TGGTGTGCGATCTGTGGGAGCCCACTAATTATTAACGCGCCGAGTGCCGGGACAATCTGTCACCGCAGCAACTAGCAATCCGTTTACAAGGCTTTAGATTGCTGGGTCTTTGCTTGCTCATGAATTGCTGATTTGTATGCGACGGTCGCGATACTACGCATGACGCGTGTAACAACCAAAAACATCATAAGACGGCAGTG
>Great-crested-grebe
CGCTGTGCTATCTGTGAGAGCCCTCTAATTATCAACGCGCCGAGTGCCGGGACGATCTGGCGCCGCGGCAACTAACAATTCATTTACAAGGCTTTAGCTTGCTGTGTTTGTGCTTGCTCATGAATCGCTGATTTGTATGTGACAGTCGCGATGGTATGCATGACGGGTTTCACAACCAAGAATATCGTAAAACCGCAGTG
>Kea
TGGTATGCGATTTGTGGGCGCCTCCTAATTGTAAATGCGCGGAGTGCCGGGACGATCTGTCGCCGCAGCAACTAGCAGTTCATTTACAAGGATTTAGTTTGCTGGGTTTGTGCTTGCTCATGAGTGGTTTATTTGTCTGCGACAGTCGCGGTAGTATGCATGACGGGTCTGACAGCCAAGAACGTCGTAAGACCGCAGTC
>Downy-woodpecker
TGGTATGTGGTCTGTGGGAGCACTATAATGCTCAACGCACCGAGTGCCGGGGCGATTTGTCGCCCAGGAAACTGGCAATTCATTTACAAGGCTCGTGATTGCTGGGTTTGTGTTTGCTAATGAATTGCTTATATGTCTGCGACAGTTGTGATACTATGCATGACGGGCTTAACAGCCAAAAACATCGTAAGACCGCTGTG
>Chuck-wills-widow
TGGTATGCGATCTATGGGAGCCCTCCAATTATCAACACGCCGAGTGTCGGGACGATCTGTCGCCGCAGCCACTAGCAATTCATTTACAAGGCTTTAGATTGCTGGGTTTATGCTTGCTCATGAATTGTTGGTTTGTATGCGACAGTCGCGATAGTATGCATGACGAGTTTAACAGTCAAGAACATCGTAAGACCGCAGTG
>Cuckoo-roller
TGGTATGCGATCTGTGGGAGCCCTCTAATTATCAACACGCCGAGTGCCGGGACGATTTGTCGCCGCAGCAACTAGCAATTCATTTACAAGGCTTGTGATTGCTGGGTTTGTGCTTGCTCGTGAATTGATGATTTGTCTGCGACAGTCGCGATAGTATGCATGACGGGTTTAACAGCCAAGAACATCGTAAGACCGCAGTG
>MacQueens-bustard
TGGTATCCGCTCTGTTGAAGCCCTGCAATTGTCAACGGGCGGAGTGCCGCGACGATCTGTCGCCGCAGCAACTAGCAATTCATTTACAATGGTTTAGATTTCTGTGTGTGTGCTTGCTCATGAATTGCTGATTTGTATGCGACAGTCGCGATAGTATGCATGACGGGTTTAACAGCCAAGAACATCGTGAGGCTGCAGTG
>Yellow-throated-sandgrouse
TGGTTTGCGATCTGTGGGAGCCCTCTGATTATCAACGCGCCGGGTGCCGGGACGTTCTGGCGCCGCACCAACTAGCAATTCATTTACAAGGCGATAGATTGCTAGATTTGTGTTTACTGATGAATTGCCGATTTGTATGCGAGAGTCGGGTTAATATGCAAGACGGGTTTAACAGCCAAGACCATCGTAAGATCGCAGTG
>Bald-eagle
TGATATGCGATCTGTGGGCGCCCTCTAATTATCAACCCGCCGAGTGCCGGGACGATCTGTCCCCGCAGCAACTAGCAATTCATTTACAAGACTTGAGATTGCTATGTTTGTGCTTGCTCATGAATTGCTGATTTGTCTGCGGCAGTCGCGATAGTATGCATAACGGGTTTAACAGCCAACAACATCGTAAGACCGCAGTG
>Sunbittern
TGGTATGAGATTTGTGGGAGCCCTCTAATTATCAACGCGCCGGGTGCCGGGATGATCTGCCGCCGCAGCAACTAGCAATTCAGTTACAGGGTTTTAGATTGCTGGGTTTGTGTTTGTCCATGAATTGCTGATTTGCATACGACAGTCGCGATAGAATACACGACGGGTTTAACAGCCAAGAATATCGTAAGACTGCAGTG
>Great-cormorant
TGGTATGCGATCTGTGGGAGCCCTCTAATTATCAATGCGCCGAGTGCCGGGACGGTCTGTCGGCGCAGCAACTACCAATTCATTTACAAGGCTTTATATCGCTGGGTTCGTGCTTGCTCATGAGTTGCTGATTTGTATGCGACAGTCGCGACAGTATGCATGAAGGGTGTAACAGCCAAGAACATCGTAAGACCGCAATG
>Medium-ground-finch
AAGTTTGCGATCTGTGGGAGCCCTATAATTATCAACGCGCCAAGAGCCGGGACGATTTGTCGCCGCAGCAACTAGCGATTCATTGACGAGTCTTTAGATTGCTGGGGTGGGGCTTGCCCATGGATTGCTGATTTGTCACCGACAGTCGGGATGGTATACATGTCGGTTTTAACAGCCGAGAACATCGTAGTATCGTAGTG
>Bar-tailed-trogon
TCGTGTGCGGTCAATGGGATCTCTCTAATTATCAACGCACCGAGTGCCGGGACGATTTATCGCCGCAGCAACGAGAAATTCCTTTACGAACCTTGTCCTTGCGGAGTTTGTGTTTGCTCATGAATAGCTGATTTGCCTGCGACAGTCGCGATAGCATGCATGACGAGTTTAACAGCCAAGAACATAGTAAGACCACAGTG
>Common-cuckoo
TGGTGTGCGATCTGTCCGAGTCCTCCAATTCTCAACACGCTGAGTGCCGGGACAATCTGTCGCCGCAGCAACTAGCAATTCATTTACAAGGCCTTAGATTGCTGCGTTTGTGCTTGCTCAAGAGTTGCTGGTTTGTATCCGAAAGTCGCAAAAGTATGCTTGACGGGTTTAACACACGAGAATAGCATAAGACCGCATTG
>Red-throated-loon
TGGTATGCGATCTGCGGGAGCCCTCTAATTATCAACGCGCCGAGTGCCGGGACGATCTGTCGCCGCAGCAACTAGCAATTCATTTACAAGACATTAGATTGCTGGGTTTGTGCTTGCTCATGAATTCCTGATTTGTATGCGACAGTCGCGATAGTATGCATGACGGGTTTAACAGCCGAGAACATCGTGAGACCGCAGTG
