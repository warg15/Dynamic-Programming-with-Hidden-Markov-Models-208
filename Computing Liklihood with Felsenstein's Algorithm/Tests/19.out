-3688.4339888776067
